<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LP-5CD212M0Y5" failures="0" tests="8" name="OG_image.ScrappersLinkedIn" time="39.274" errors="1" timestamp="2025-07-22T12:05:37 IST" skipped="0">
  <testcase classname="OG_image.ScrappersLinkedIn" name="sendEmail" time="4.596"/>
  <testcase classname="OG_image.ScrappersLinkedIn" name="LinkedInPostInception" time="1.893"/>
  <testcase classname="OG_image.ScrappersLinkedIn" name="LinkedInPostInception" time="1.830"/>
  <testcase classname="OG_image.ScrappersLinkedIn" name="logLastLine" time="0.007"/>
  <testcase classname="OG_image.ScrappersLinkedIn" name="afterSuite" time="1.485">
    <error message="Failed to open file:/C:/Users/v-rkammili/OneDrive%20-%20Microsoft/Automation_Projects/Scrapper_autoamtion/Scrapper_autoamtion/AllTests.html. Error message: The system cannot find the file specified.
" type="java.io.IOException">
      <![CDATA[java.io.IOException: Failed to open file:/C:/Users/v-rkammili/OneDrive%20-%20Microsoft/Automation_Projects/Scrapper_autoamtion/Scrapper_autoamtion/AllTests.html. Error message: The system cannot find the file specified.

at java.desktop/sun.awt.windows.WDesktopPeer.ShellExecute(WDesktopPeer.java:115)
at java.desktop/sun.awt.windows.WDesktopPeer.browse(WDesktopPeer.java:101)
at java.desktop/java.awt.Desktop.browse(Desktop.java:533)
at OG_image.BaseTest.afterSuite(BaseTest.java:99)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:386)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- afterSuite -->
  <testcase classname="OG_image.ScrappersLinkedIn" name="LinkedInPostInception" time="1.708"/>
  <testcase classname="OG_image.ScrappersLinkedIn" name="LinkedInPostInception" time="1.835"/>
  <testcase classname="OG_image.ScrappersLinkedIn" name="LikedInLanding" time="25.920"/>
</testsuite> <!-- OG_image.ScrappersLinkedIn -->
